lib/Target/TTNN/TTNNToFlatbuffer.cpp

```diff
@@ -258,6 +260,9 @@ emitTTNNOperation(FlatbufferObjectCache &cache, Operation *op,
   if (auto sqrtOp = dyn_cast<SqrtOp>(op); sqrtOp) {
     return createOperation(cache, createEltwiseOp(cache, sqrtOp), debugString);
   }
+  if (auto exponentialOp = dyn_cast<ExponentialOp>(op); exponentialOp) {
+    return createOperation(cache, createEltwiseOp(cache, exponentialOp), debugString);
+  }
```

Comment:
Btw you also need to make changes in `runtime/lib/ttnn/program.cpp`, in function:
```
static void
run(::tt::target::ttnn::EltwiseOp const *op, ...
```

In GH-527 I've made a small refactoring of that function, but you can just add duplicated code as was previously done and then I will pick up those changes in my PR if you plan to merge before me.